TYPO3.Flow/Classes/TYPO3/Flow/I18n/Cldr/CldrRepository.php

```diff
@@ -49,15 +52,15 @@ class CldrRepository
      * reside and 'locale' is used to define which files are included in the
      * relation (e.g. for locale 'en_GB' files would be: root + en + en_GB).
      *
-     * @var array<\TYPO3\Flow\I18n\Cldr\CldrModel>
+     * @var array<CldrModel>
      */
     protected $models;
 
     /**
-     * @param \TYPO3\Flow\I18n\Service $localizationService
+     * @param Service $localizationService
```

Comment:
Or maybe we should just also rename that class alltogether, the name "Service" is really bad after all and even doesn't match the naming scheme in other parts (ie. "Object\ObjectManager", "Reflection\ReflectionService", etc.)